Mathlib/Topology/Separation.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro
 -/
+import Mathlib.Topology.Compactness.Lindelof
```

Comment:
Can you add in the doc some references to Lindelöf spaces and to the way some separation axioms are deduced through the theory of Lindelöf spaces?